spreg/sur.py

```diff
@@ -676,34 +695,34 @@ class ThreeSLS(BaseThreeSLS, REGI.Regimes_Frame):
     dictionaries for Y and X, and sur_dictZ for endogenous variables (yend) and
     additional instruments (q).
 
-    >>> bigy,bigX,bigyvars,bigXvars = pysal.spreg.sur_utils.sur_dictxy(db,y_var,x_var)
-    >>> bigyend,bigyendvars = pysal.spreg.sur_utils.sur_dictZ(db,yend_var)
-    >>> bigq,bigqvars = pysal.spreg.sur_utils.sur_dictZ(db,q_var)
+    >>> bigy,bigX,bigyvars,bigXvars = spreg.sur_dictxy(db,y_var,x_var)
+    >>> bigyend,bigyendvars = spreg.sur_dictZ(db,yend_var)
+    >>> bigq,bigqvars = spreg.sur_dictZ(db,q_var)
```

Comment:
see "spaces" comment above for formatting recommendation of these three variable declarations.